.github/workflows/consistency-test.yml

```diff
@@ -0,0 +1,77 @@
+name: consistency_testing
+on:
+  repository_dispatch:
+    types: [chaos-test-command]
+
+jobs:
+  run_consistency_tests:
+    runs-on: ubuntu-20.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      CI: 1
+      CI_INSTALL_INFRA: 1
+      AWS_EC2_METADATA_DISABLED: true
+
+    steps:
+      - name: Setup gcloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          project_id: 'redpandaci'
+          service_account_key: ${{ secrets.GCR_JSON}}
+          export_default_credentials: true
+
+      - name: setup env slash command
+        run: |
+          echo "Populating env from PR context"
+          commit=${{ github.event.client_payload.pull_request.head.sha }}
+          short=${commit:0:7}
+
+          # Make env available to next steps
+          echo "deb_pkg_url=gs://vectorizedio/rp_dev/release/clang/$short/redpanda_*.deb" >> $GITHUB_ENV
+          echo "upload_dir=redpanda/pr-${{ github.event.client_payload.pull_request.number }}" >> $GITHUB_ENV
+        if: ${{ github.event.action == 'chaos-test-command' }}
+
+      - name: fetch release from gcb
+        run: |
+          gsutil cp $deb_pkg_url redpanda.deb
```

Comment:
maybe check if the package exists first? And then print message and fail explicitly when that's the case?